cypress/e2e/signIn.cy.js

```diff
@@ -2,6 +2,25 @@
 
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
+    cy.visit('https://react-redux.realworld.io');
 
+    cy.get(':nth-child(2) > .nav-link').click();
+
+    cy.get('.navbar-brand').should('exist');
```

Comment:
There is no need to assert the existence of every element on the page. Focus on the elements that are relevant to the specific functionality that you are testing.